blog/2023-06-23-newsletter.mdx

```diff
@@ -0,0 +1,164 @@
+description: |
+    Recent and upcoming activity in the conda community.
+authors: [tnabtaf, pinak-datta]
+tags: [conda, newsletter]
+image: img/blog/2023-06-23-newsletter/balloon-launch.jpg
+---
+
+import Image from '@theme/IdealImage';
+import launch from '@site/static/img/blog/2023-06-23-newsletter/balloon_launch.jpg';
+import scipybanner from '@site/static/img/blog/2023-06-23-newsletter/scipy_banner.png';
+import packagingcon from '@site/static/img/blog/2023-06-23-newsletter/packagingcon.jpg';
+import rattler from '@site/static/img/rattler.png';
+import youtube from '@site/static/img/blog/2023-06-23-newsletter/youtube.png';
+import cf from '@site/static/img/conda_forge_wide.png';
+import bioconda from '@site/static/img/bioconda_word_logo.png';
+import forum from '@site/static/img/blog/2023-06-23-newsletter/conda_community_forum.png';
+
+<Image img={launch} alt="Banner image for blog post; The Conda Newsletter is launching - hot air balloon inflating"/>
+
+<p>
+    <i>
+        <small>
+            Image credit: <a href="https://unsplash.com/photos/sN40hKv8_cI">Road Trip with Raj on Unsplash</a>
+        </small>
+    </i>
+</p>
+
+**Welcome to the first ever conda community newsletter!** The newsletter will highlight recent and upcoming activity that is relevant to the [conda community](https://conda.org/). This first issue highlights [community gathering spaces](#community-gathering-spaces), [upcoming events](#upcoming-events), the new [rattler library](#rattler), [channel updates from conda-forge and Bioconda](#community-channel-updates), a [summary or recent conda-related web content](#conda-on-the-web), and finally [releases and announcements](#releases--announcements).
+
+We are hoping these will become a semi-regular feature in the conda landscape going forward. If you have an item to include in a future newsletter, please let us know by posting it to the [Conda Communications Matrix Channel](https://matrix.to/#/%23conda-communications:matrix.org) or by creating an issue in the [Conda Communications GitHub Repo](https://github.com/conda/communications/issues).
+
+<!-- truncate -->
+
+Finally, if you want to be notified about these newsletters from the comfort of your inbox, **please sign up for the [Conda Announce Mailing list](https://lists.conda.org/wws/info/announce).**
+
+Happy reading,<br />
+The Editors
+
+---
+
+## Community Gathering Spaces
+
+It has been a busy year for creating and consolidating online gathering spaces for the conda community. We now have a community web site, a Q&A/forum stie, a unified chat platform, a mailing list, and more. See **[Pinak Datta's recent post](/blog/2023-06-22-conda-spaces-blogpost)** for a complete list and links.
+
+### Community calls
+
+Please join us every other week to learn what is going on in the conda community and to provide your input.
+
+* The Mamba project has [bi-weekly community calls](https://hackmd.io/iOgkxNMpTea6lWczjCov9Q?view) on Tuesdays. Our next call is June 27.
```

Comment:
Our next call is "on" June 27 would be better I suppose.

Reply:
Also fixed.